internal/ui/backup/progress.go

```diff
@@ -84,191 +82,165 @@ func NewProgress(printer ProgressPrinter, interval time.Duration) *Progress {
 // Run regularly updates the status lines. It should be called in a separate
 // goroutine.
 func (p *Progress) Run(ctx context.Context) {
-	var (
-		lastUpdate       time.Time
-		total, processed Counter
-		errors           uint
-		started          bool
-		currentFiles     = make(map[string]struct{})
-		secondsRemaining uint64
-	)
-
-	t := time.NewTicker(time.Second)
-	signalsCh := signals.GetProgressChannel()
-	defer t.Stop()
 	defer close(p.closed)
 	// Reset status when finished
 	defer p.printer.Reset()
 
+	if p.interval == 0 {
+		return
```

Comment:
Won't this prevent the `signalsCh` from working when there's no periodic status update?

Reply:
Right. Fixed.